### ISSUE TITLE
Test randomly failling

### ISSUE THREAD

**Issue by ph** [logstash-plugins/logstash-input-kafka#118]
The current build is failling on travis, Probably some test are flaky or the travis box is a bit saturated.

```
Failures:
  1) inputs/kafka #kafka-decorate should show the right topic and group name in decorated kafka section
     Failure/Error: wait(timeout_seconds).for { queue.length }.to eq(num_events)
       expected: 103
            got: 0
       (compared using ==)
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:13:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:10:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:9:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:28:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./spec/integration/inputs/kafka_spec.rb:93:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
  2) inputs/kafka #kafka-topics-pattern should consume all messages from all 3 topics
     Failure/Error: wait(timeout_seconds).for { queue.length }.to eq(3*num_events)
       expected: 309
            got: 206
       (compared using ==)
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:13:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:10:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:9:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:28:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./spec/integration/inputs/kafka_spec.rb:74:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
  3) inputs/kafka #kafka-topics should consume all messages from plain 3-partition topic
     Failure/Error: wait(timeout_seconds).for { queue.length }.to eq(num_events)
       expected: 103
            got: 0
       (compared using ==)
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:13:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:10:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/handler.rb:9:in `handle_matcher'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:28:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:44:in `with_wait'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait/target.rb:30:in `to'
     # ./spec/integration/inputs/kafka_spec.rb:35:in `(root)'
     # ./vendor/bundle/jruby/1.9/gems/rspec-wait-0.0.9/lib/rspec/wait.rb:46:in `(root)'
Finished in 6 minutes 2 seconds (files took 3.12 seconds to load)
5 examples, 3 failures
Failed examples:
rspec ./spec/integration/inputs/kafka_spec.rb:88 # inputs/kafka #kafka-decorate should show the right topic and group name in decorated kafka section
rspec ./spec/integration/inputs/kafka_spec.rb:69 # inputs/kafka #kafka-topics-pattern should consume all messages from all 3 topics
rspec ./spec/integration/inputs/kafka_spec.rb:30 # inputs/kafka #kafka-topics should consume all messages from plain 3-partition topic

Randomized with seed 41531

```

**Comment by original-brownbear** [comment]
same flakiness locally for ITs ... investigating

**Comment by original-brownbear** [comment]
fixed in #197
